.github/workflows/ci.yml

```
@@ -17,13 +17,10 @@ jobs:
       - name: Setup Cue
         uses: cue-lang/setup-cue@v1.0.0
         with:
-          cue-version: 0.11.0
-      
-      - name: Validate our security-insights.yml
-        run: cue vet .github/security-insights.yml schema.cue
+          version: 0.11.0
```

```suggestion
          cue-version: 0.11.0
```

Welp, that didn't work. Will look more closely after my next flight...